src/compiler/emitter.ts

```diff
@@ -3,6 +3,190 @@
 
 /* @internal */
 namespace ts {
+    export const defaultInitCompilerOptions: CompilerOptions = {
+        module: ModuleKind.CommonJS,

```

Comment:
Don't you think it should be `commonjs` since it is the most common? I think most project is either commonjs or amd.